tests/integration/test_integration.py

```diff
@@ -997,13 +998,12 @@ def test_filter_log_entries_by_list(self, admin_client, list_a, list_b, paged):
     def test_filter_log_entries_by_list_is_ordered(self, admin_client, list_a, list_b):
         criterion = AuditLogSearchCriterion(filter_record_lists=[list_b.identifier])
         results = list(admin_client.search_for_audit_log_entries(criterion))
-        assert len(results) == 4
+        assert len(results) == 3
         for event in results:
             assert event.list_identifier == list_b.identifier
         expected_actions = [
-            AuditLogAction.LISTAWAITINGAPPROVALREMOVED,
             AuditLogAction.LISTPUBLISHED,
```

Comment:
These are the same 3 actions that appear in the Favorites app in the latest MI build